pt-br/ruby-pt.html.markdown

```diff
@@ -6,6 +6,7 @@ contributors:
   - ["Bruno Henrique - Garu", "http://garulab.com"]
   - ["Jean Matheus Souto", "http://jeanmatheussouto.github.io"]
 translators:
+  - ["Alan Peterson Carvalho Silva", "https://twitter.com/DemonKart"]
```

Comment:
if you're gonna add yourself as a translator, please don't add yourself as the first one